src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -118,6 +118,7 @@ Copyright (c) .NET Foundation. All rights reserved.
             ReferenceAssemblyPaths="@(ReferencePath)"
             RootAssemblyNames="@(TrimmerRootAssembly)"
             TrimMode="$(TrimMode)"
+            DefaultAction="$(_TrimmerDefaultAction)"
```

Comment:
Should `_TrimmerDefaultAction` use underscore? This is the only property the developers would need to change to make their apps fully linker, right?

Reply:
I would like it to be public - but in the discussions about a similar [`TrimAllAssemblies`](https://github.com/mono/linker/blob/main/docs/design/trimmed-assemblies.md#trimallassemblies-global-opt-in) opt-in there were concerns about making it too easy to trim everything. I restarted the discussion in https://github.com/dotnet/sdk/issues/16140.